nighthawk/test/test_data/benchmark_http_client_test_envoy.yaml

```diff
@@ -0,0 +1,89 @@
+admin:
+  access_log_path: /dev/null
+  address:
+    socket_address:
+      address: 127.0.0.1
```

Comment:
Yeah, I reckon the IPv6 issue will also arise here.

Reply:
I switched this to `{{ ip_loopback_address }}`, I think that is the the right thing to do here?